package/koreader/koreader.draft

```diff
@@ -0,0 +1,3 @@
+name=KOReader
+desc=ebook reader
+call=/opt/bin/koreader
```

Comment:
```suggestion
call=/opt/bin/koreader
term=killall -9 luajit
```

Reply:
Running this change on oxide showed a me for the first 3 times a crash screen for a short while and made koreader restart before I could actually quit by holding the left button.

Reply:
Image of crash screen (only shown for about 1 second):
![20200923_004209_lq](https://user-images.githubusercontent.com/22298664/93944753-d52fe000-fd35-11ea-8b3e-7e7197ae26f8.jpg)

(I also verified that my `/opt/etc/draft/koreader.draft` actually contains the `term=` entry from above.)

Reply:
Oh right, the script needs to be killed as well. Forgot to test with just draft. There is a variable you can pass into KOReader to keep it from restarting itself, but that wont play nice with oxide.